models/user/userValidation.js

```diff
@@ -0,0 +1,27 @@
+import Joi from 'joi';
+import {
+  phoneNumberPattern,
+  addressValidationSchema,
+  idValidationSchema,
+  passwordPattern,
+} from '../../helpers/index.js';
+import { roles, accountStatuses } from '../../constants/index.js';
+import { orderItemValidationSchema } from '../order/order.validation.js';
+
+export const userValidationSchema = Joi.object({
+  fullName: Joi.string().min(3).required(),
```

Comment:
more common approach is to use `firstName` and `lastName`, jfyi

Reply:
Thanx, I changed it

Reply:
I don't see any changes, it is still `fullName`)
anyway, you should stick with logic you all agreed to, this is just an information for you

Reply:
I added a comment here because this conversion is no longer displayed in the "File changes" for the last commit